docker/cachet.env

```diff
@@ -0,0 +1,44 @@
+APP_ENV=-production
+APP_DEBUG=false
+APP_URL=
+APP_LOG=errorlog
+APP_KEY=base64:FDQbZbYTpw910TM2sycLTqknNIVeEZkIEXDX/ESdom4=
+
+DB_DRIVER=pgsql
+DB_HOST=postgres
+DB_DATABASE=scratch
```

Comment:
I think we should gitignore this file and only commit `cachet.env.example` to version control, for consistency. This way we can remove the credentials like in the other files in this folder.